src/components/sidebar/sidebar.component.tsx

```diff
@@ -96,6 +96,8 @@ export interface SidebarProps
    * @internal
    * Sets className for component. INTERNAL USE ONLY. */
   className?: string;
+  /** @private @ignore */
```

Comment:
suggestion (non-blocking): whilst this is a private prop it's probably worth adding a description above it for sake of maintenance etc